src/styles/variables.module.scss

```diff
@@ -1,7 +1,11 @@
 $--header-height: 80;
-$--footer-height: 50;
-$--main-offset: $--header-height + $--footer-height + 2;
+$--header-height-small: 50;
+$--footer-height: 40;
+$--main-offset-borders-height: 2;
+$--main-offset: $--header-height + $--footer-height + $--main-offset-borders-height;
+$--main-offset-small: $--header-height-small + $--footer-height + $--main-offset-borders-height;
 
 :export {
   mainOffset: $--main-offset;
+  mainOffsetSmall: $--main-offset-small;
```

Comment:
What is small? tablet? mobile? please write the appropriate breakpoint.

Reply:
the '$--header-height' is for width 1280 and above. breakpoint DESKTOP.
and the '$--header-height-small' is for all the rest.

Reply:
So why not to change `$--header-height` to `$--header-height-desktop` and `$--header-height-small` to `$--header-height`???